identity-iota/src/tangle/traits.rs

```diff
@@ -18,7 +18,8 @@ pub trait TangleRef {
   fn set_previous_message_id(&mut self, message_id: MessageId);
 }
 
+// TODO: remove TangleResolve with ClientMap refactor?
```

Comment:
I think it still needs to be a trait since the (unused) spec-compliant resolver and more importantly presentation and credential validators use it. Not sure if it can be removed entirely to be honest.

Reply:
Okay, shame. Maybe there are workarounds I haven't found yet.